app/styles/module/_ads.scss

```diff
@@ -145,127 +145,23 @@ $closeButtonHeight: 40px;
   border: 0;
 }
 
-.invisible-high-impact-wrapper:not(.out-of-page-template-loaded) {
-  align-items: center;
-  display: flex;
-  justify-content: center;
-  position: fixed;
-  width: 100%;
-  z-index: $z-modal-dialog;
-
-  .close {
-    color: $wds-color-white;
-    cursor: pointer;
-  }
-
-  .invisible-high-impact-2,
-  .floor-adhesion iframe {
-    display: block;
-    margin: 0 auto;
-    position: relative;
-  }
-
-  &.floor-adhesion {
-    bottom: 0;
-    // needed for Safari 9 - it doesn't position elements from the left by default
-    left: 0;
-
-    .background {
-      background: rgba(0, 0, 0, .8);
-      height: 100%;
-      // needed for Safari 9 - it doesn't position elements from the left by default
-      left: 0;
-      position: absolute;
-      width: 100%;
-    }
-
-    .label {
-      display: none;
-    }
-
-    .close {
-      align-items: center;
-      background-color: #000;
-      border: 2px solid white;
-      border-radius: 50%;
-      display: flex;
-      height: $closeButtonHeight;
-      justify-content: center;
-      padding: 5px;
-      position: absolute;
-      right: 0;
-      top: -($closeButtonHeight / 2);
-      width: $closeButtonHeight;
-      //This is needed to make sure that close button is always displayed above ad
-      z-index: $z-modal-dialog;
-    }
-  }
-
-  &.interstitial {
-    height: 100%;
-    left: 0;
-    top: 0;
-
-    .provider-container {
-      margin: 0;
-    }
-
-    .background {
-      background: rgba(0, 0, 0, .9);
-      bottom: 0;
-      height: 100%;
-      left: 0;
-      position: absolute;
-      width: 100%;
-    }
-
-    .top-bar {
-      background-color: rgba(0, 0, 0, .5);
-      display: flex;
-      flex-direction: row-reverse;
-      justify-content: space-between;
-      left: 0;
-      position: absolute;
-      top: 0;
-      width: 100%;
-      //This is needed to make sure that close button is always displayed above ad
-      //ad can take the whole space of interstitial if too big creative is targeted
-      //- it shouldn't happen but it protects users from being not able to close interstitial
-      z-index: $z-modal-dialog;
-    }
-
-    .close {
-      padding: 20px;
-    }
-
-    .label {
-      color: #d5d4d4;
-      font-size: 17px;
-      padding: 20px 10px;
-    }
-  }
-}
-
 .out-of-page-template {
   margin: 0;
 
   &.floor-adhesion {
-    padding: 20px 0;
+    padding: 20px 0 0 0;
 
     .button-close {
-      align-items: center;
-      background-color: #000;
       border: 2px solid white;
       border-radius: 50%;
-      display: flex;
       height: $closeButtonHeight;
-      justify-content: center;
-      padding: 5px;
-      position: absolute;
-      right: 0;
       top: -($closeButtonHeight / 2);
       width: $closeButtonHeight;
     }
+
+    .advertisement-label {
```

Comment:
Is it needed? It seems it's added in the ad-engine styles

Reply:
Good point - removed.